bentoml/_internal/runner/utils.py

```diff
@@ -17,11 +17,13 @@
 _drv = None
 
 if TYPE_CHECKING:
-    from _ctypes import _SimpleCData
+    from ctypes import _PointerLike  # type: ignore[reportPrivateUsage]
+    from ctypes import _SimpleCData  # type: ignore[reportPrivateUsage]
```

Comment:
pyright cannot recognize `_ctypes` imports since `_ctypes` is a compiled C binary